### PR TITLE
Implementation of MooTV web application

### PR DESCRIPTION
**Pull request by why-a-en**
###  MooTV features ✔️ 

- This branch is a collection of all the features that build up the MooTV website.
- The MooTV website is currently a single-page application built using webpack, and linters.
- The main layout is implemented using the flexbox with appropriate color swatches.
- The comment popup box is added and its API data are manipulated through JavaScript dynamically.
- We used TVmaze API and involvement API in this website.
- The fetched data from TVmaze API is used to display all the images, descriptions, and genres.
- The involvement API is used to make the like and comment buttons interactive.
- Two counter functions are tested using the jest library with mock files
- Added project presentation video to README file
- Added distribution code and will be deployed after merging to the main branch

### CODE REVIEW

File: src/modules/commentFunctions.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            arr.forEach((item) => {
+              createComment(item);
+              commentsCounter();
+            });
+          }
+        });
+      });
+    }
+  };
+  submitCommentBtn.addEventListener('click', submitCommentHandler);
```
Comment (Rustamxon7) [inline src/modules/commentFunctions.js:53]: You guys did a great job! Perfectly implemented code!

- [ ] All loading movies, likes are well-implemented. However, I got one issue in your comments section. Let me explain, what I'm getting. First I opend Batman movie then left a comment in it. Then I tried to open other movie but somehow the comment that I left in Batman is visible in other movie as well. Also, the comments are keeps adding from other movies comments. Please, fix the issue. Thank you!

Here is the video about the issue:

https://user-images.githubusercontent.com/69011963/233781228-d7838704-94db-4416-89ee-e97330f2d268.mov

File: README.md [hunk 2]
```diff
@@ -1 +1,156 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live demo <a name="live-demo"></a>
+
+- [Video Presentation](https://drive.google.com/file/d/1-YtxC95Cn8p672IHB-ZiY4i7h8Eai5dS/view?usp=sharing)
+
+- [Live Demo](https://anbehindy.github.io/JS-Capstone/dist)
```
Comment (Rustamxon7) [inline README.md:64]: [OPTIONAL] It looks like your live demo is not deployed as expected. Kindly add correct demo link or just add message about link is not available right now.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/commentFunctions.js (hunk 1 above)
Comment (anbehindY) [reply src/modules/commentFunctions.js:53]: Thanks for the review
Issue fixed

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**